api/provider/aws/test_aws/environment_create_test.go

```diff
@@ -261,30 +260,6 @@ func TestEnvironmentCreateDefaults(t *testing.T) {
 		AuthorizeSecurityGroupIngress(gomock.Any()).
 		Return(&ec2.AuthorizeSecurityGroupIngressOutput{}, nil)
 
-	// ensure we pass the default instance type, ami id, and user data to the launch configuration
```

Comment:
why is this being removed?

Reply:
Because of a failing unit test related to work being done in another branch off api-refactor. Specifically, the default behavior (in `api-refactor) of creating an environment (i.e. `l0 environment create env_a`) is creating a stateless environment. This means that Launch Configurations and Auto Scaling Groups are not created. I'd rather not have a commit that shows breaking unit tests and would rather fix the unit tests that are breaking given `api-refactor`'s current state.

Reply:
i see, make note that this will cause issue with https://github.com/quintilesims/layer0/pull/567, since it should exist in that pr